### PR TITLE
Additional component features

### PR DESCRIPTION
**Pull request by DavidCramer**
Adds simple conditional logic
Collapsible groups
Extendable blueprint
fixed value sanitisation

### CODE REVIEW

**Review by pereirinha — COMMENTED** [review]
@DavidCramer 

Looks good. Just left you a comment.

File: php/ui/component/class-on-off.php [hunk 1]
```diff
@@ -34,7 +54,7 @@ protected function input( $struct ) {
 		$struct['attributes']['type']  = 'checkbox';
 		$struct['attributes']['name']  = $this->get_name();
 		$struct['attributes']['id']    = $this->setting->get_slug();
-		$struct['attributes']['value'] = true;
+		$struct['attributes']['value'] = 'true';
```
[the remark this answers is not in the document]
Comment (DavidCramer) [reply php/ui/component/class-on-off.php:57]: I have no idea 😂

I'll fix it.

[the remark this answers is not in the document]
Comment (DavidCramer) [reply php/ui/component/class-on-off.php]: @pereirinha - fixed. I changed it to a string to debug something, and forgot to revert it back. All good now though.

**Review by pereirinha — APPROVED** [review]
Thanks… looks good.

**Pull request merged**